app/bundles/EmailBundle/Model/EmailActionModel.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Mautic\EmailBundle\Model;
+
+use Mautic\CategoryBundle\Entity\Category;
+
+readonly class EmailActionModel
+{
+    public function __construct(
+        private EmailModel $emailModel
+    ) {
+    }
+
+    public function setCategory(array $emailsIds, Category $newCategory): array
+    {
+        $emails = $this->emailModel->getRepository->findBy(['id' => $emailsIds]);
```

Comment:
```suggestion
        $emails = $this->emailModel->getRepository()->findBy(['id' => $emailsIds]);
```
@andersonjeccel my bad. I think that this PR is still work in progress. Wait for the CI checks to be green to retest.

Reply:
Got it!
Thanks for replying